scripts/semconv/templates/SemanticAttributes.ts.j2

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+{%- macro print_value(type, value) -%}
+  {{ "\"" if type == "string"}}{{value}}{{ "\"" if type == "string"}}
+{%- endmacro %}
+{%- macro upFirst(text) -%}
+  {{ text[0]|upper}}{{text[1:] }}
+{%- endmacro %}
+{%- macro lowerFirst(text) -%}
+  {{ text[0]|lower}}{{text[1:] }}
+{%- endmacro %}
+
+// DO NOT EDIT, this is an Auto-generated file from scripts/semconv/templates/{{template}}
+export const {{class}} = {
+  {%- for attribute in attributes | unique(attribute="fqn") %}
+
+  /**
+  * {% filter escape %}{{attribute.brief | to_doc_brief}}.{% endfilter %}
+    {%- if attribute.note %}
+  *
+  * Note: {% filter escape %}{{attribute.note | to_doc_brief}}.{% endfilter %}
+    {%- endif %}
+    {%- if attribute.deprecated %}
+  *
+  * @deprecated {{attribute.deprecated | to_doc_brief}}.
+    {%- endif %}
+  */
+    {%- if attribute.deprecated %}
+  @Deprecated
+    {%- endif %}
+  {{attribute.fqn | to_const_name}}: '{{attribute.fqn}}',
+  {%- endfor %}
+
+  {%- if class == "SemanticAttribute" %}
+
+  // Manually defined and not YET in the YAML
```

Comment:
is there plan to add them upstream ?

Reply:
Good question, I am not sure. Might be worth considering. I have added them to avoid needing to change too much code

Reply:
Most of these sound like obsolete attributes that are represented/named differently

- `grpc.kind` should be represented as the Span Kind
- `grpc.status_code` should be `rpc.grpc.status_code`
- `grpc.error_name` is redundant with the code
- `gprc.status_message` is probably worth filing a spec PR for (as `rpc.grpc.status_message`). The span status message might not be ideal to use for it, for the same reason we don't mark spans as failed in RecordException.
- `http.status_text` was deemed useless and removed without replacement, see https://github.com/open-telemetry/opentelemetry-specification/issues/950.
- `http.error_name`, `http.status_text`: No idea what these are.

Reply:
I think they shouldn't be defined here anyway if they aren't in the semantic conventions. Anything in this package should be only things approved by spec and by including them we are giving them our implicit approval for third parties. Most people will assume anything they find in autocomplete here is "official"